src/components/CreateResetPassword/components/SuccessPage/SuccessPageCtrl.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import {
+  withRouter,
+  Redirect,
+} from 'react-router-dom';
+
+import SuccessPage from './SuccessPage';
+
+class SuccessPageCtrl extends Component {
+    state = {
+      isRedirectButtonClicked: false,
+    };
+
+    handleRedirectButtonClick = () => {
+      this.setState({ isRedirectButtonClicked: true });
+    };
+
+    render() {
+      const { isRedirectButtonClicked } = this.state;
+
+      if (isRedirectButtonClicked) {
+        return <Redirect
```

Comment:
this might be a code style issue - `return (` the only thing should be on this line